model_management.py

```diff
@@ -0,0 +1,95 @@
+import config_manager
+import project_logger
+import gymnasium as gym
+
+from typing import Union, Optional
+from stable_baselines3 import DQN, PPO
+
+
```

Comment:
we need to decide on a convention and use the var names as follows:

- agent = DQN / PPO
- model - will be the neural network

so in this file - in every place where it is written model it should be replaced by agent